img/b/bitget.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#0088E5" d="M2.2 9.2c1.7-1 3.5-1.7 5.2-2.4 3-1.3 5.9-2.5 8.9-3.6.6-.2 1.6-.4 1.7.4 0 1.3-.3 2.5-.4 3.7l-1.4 8c-.1.8-1.2 1.3-2 .7l-5-3.4c-.6-.6 0-1.4.4-1.8 1.4-1.4 3-2.6 4.2-4 .4-.9-.7-.1-1 0-2 1.4-3.8 2.8-5.8 4-1 .5-2.2 0-3.3-.3-.9-.4-2.3-.7-1.5-1.3Z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#00e0ff" viewBox="0 0 23.73 25.52"><path d="M10.56 9.16h6.35l6.5 6.46c.42.42.42 1.1 0 1.53l-8.34 8.38H8.53l1.98-1.92 7.26-7.22-7.17-7.22m2.57 7.19H6.82L.32 9.9c-.42-.42-.42-1.1 0-1.52L8.65 0h6.54l-1.98 1.92-7.26 7.22 7.17 7.22"/></svg>
```

Reply:
Thank you for your modification, but may I ask what is the basis for your modification? ❤️ @fletcherdodd

Reply:
No worries @Yang-Nankai! As per [the guidelines, point 2](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) talks about optimising your file as much as possible to reduce the file size. I usually run my SVG files through 3 different optimisation tools (a couple are mentioned in contributing doc) just to ensure it as small as possible. 😇